pyproject.toml

```diff
@@ -25,13 +25,13 @@ classifiers = [
     "Framework :: AiiDA"
 ]
 keywords = ["aiida", "plugin", "aurora"]
-requires-python = ">=3.8"
+requires-python = ">=3.9"
 dependencies = [
     "aiida-core>=2.0,<3",
     "aiida-ssh2win>=0.2",
     "dgbowl-schemas>=111,<=113",
     "pydantic<2.0",
-    "matplotlib",
+    "scipy~=1.11.2",
```

Comment:
Not really important, but this version requirement for scipy seems quite strict.

Reply:
I suppose so 😅 I really only need `cumtrapz`. Let me find the minimum `scipy` version that includes a reasonable implementation of the function.

Reply:
Hmm, the function became available in 0.12. However, I've had difficulties installing versions less than 1. How about `scipy>=1.0,<2`? This should ensure support flexibly, while avoiding future breaking changes.

Reply:
Yes, something like that would be good. Just make sure that it works with scipy 1.0

Reply:
So I tried installing 1.0. Same issues as with <1. The earliest I am able to install is 1.5.4, which works for the app. Pinning `scipy>=1.5.4,<2`